xpublish/rest.py

```diff
@@ -120,12 +166,83 @@ def serve(self, host='0.0.0.0', port=9000, log_level='debug', **kwargs):
         log_level : str
             App logging level, valid options are
             {'critical', 'error', 'warning', 'info', 'debug', 'trace'}.
-        kwargs :
+        **kwargs :
             Additional arguments to be passed to :func:`uvicorn.run`.
 
         Notes
         -----
         This method is blocking and does not return.
 
         """
-        uvicorn.run(self.app, host=host, port=port, log_level=log_level, **kwargs)
+        uvicorn.run(
+            self.app, host=host, port=port, log_level=log_level, **kwargs
+        )  # pragma: no cover
```

Comment:
@benbovy and/or @lsetiawan  - if either of you have ideas how we could test this line, I'm happy to remove the pragma.

Reply:
Maybe something like this would work?

```python
from unittest import mock

def test_serve(airtemp_rest):
    with mock.patch('uvicorn.run'):
        airtemp_rest.serve(...)
        uvicorn.run.assert_called_once_with(...)
```

Reply:
Great idea. Done in b47e4dc.